sections/call-to-action.liquid

```diff
@@ -149,6 +152,14 @@
         "id": "image",
         "label": "Background image"
       },
+      {
+        "type": "number",
+        "id": "overlay_opacity",
+        "label": "Overlay percentage",
```

Comment:
Can you please rename the label to "Overlay opacity"?

Reply:
I don't think we should do that because once Herman asked to rename the label `overlay opacity` to `overlay percentage` and here I stick to one style. Here is an example: 
![Teampaper_2024-01-22 13-54-03@2x](https://github.com/booqable/tough-theme/assets/40244261/55433474-4e12-4fb5-81ef-8d4aa3614171)

Reply:
Ok, good to know 👍